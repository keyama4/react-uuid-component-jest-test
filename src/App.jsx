import React from 'react';
import './App.css';
import { v4 as uuid } from 'uuid';

function App() {
  return (
    <div className="App">
      <p>{uuid()}</p>
    </div>
  );
}

export default App;
