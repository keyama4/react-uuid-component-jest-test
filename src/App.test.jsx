import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

jest.mock('uuid',() => ({
  v4: () => '12fa6ae8-75d3-40b3-97af-172d0a304d37',
}));

test('renders currently', () => {
  const {container} = render(<App />);
  expect(container).toMatchSnapshot();
});
