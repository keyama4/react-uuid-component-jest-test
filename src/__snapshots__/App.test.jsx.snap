// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders currently 1`] = `
<div>
  <div
    class="App"
  >
    <p>
      12fa6ae8-75d3-40b3-97af-172d0a304d37
    </p>
  </div>
</div>
`;
